source/engine/serialize/ComponentSerialization.cpp

```diff
@@ -195,25 +195,26 @@ auto DeserializePhysicsMaterial(std::istream& stream, const DeserializationConte
     return physics::PhysicsMaterial{friction, restitution};
 }
 
-void SerializePointLight(std::ostream& stream, const graphics::PointLight& out, const SerializationContext& ctx, const std::any&)
+void SerializePointLight(std::ostream& stream, const graphics::PointLight& out, const SerializationContext&, const std::any&)
```

Comment:
Without the entity, serialize/deserialize can just be done directly for both `PointLight` and `SpotLight`:
```cpp
serialize::Serialize(stream, out);
```
and
```cpp
auto out = ...
serialize::Deserialize(stream, out);
return out;
```